app/bundles/DashboardBundle/Translations/en_US/messages.ini

```diff
@@ -12,6 +12,8 @@ mautic.dashboard.label.lang="Lang"
 mautic.dashboard.label.title="Title"
 mautic.dashboard.menu.index="Dashboard"
 mautic.dashboard.update.past.tense="updated"
+mautic.dashboard.timezone="Based on system timezone"
+mautic.dashboard.timezone.help="The timezone for all dashboard widgets is defined in the system settings."
```

Comment:
Are you sure it cannot be overwritten by the timezone set on user profile?

Reply:
Tested here using production and dev environments, all widgets follow only the system timezone
It would be great if widgets considered the user timezone

My settings on production env 5.1.0 (UTC-3)
![image](https://github.com/user-attachments/assets/af3286d6-041f-440c-8299-d2225f399da8)

The widget (UTC+2)
![image](https://github.com/user-attachments/assets/caa5860c-e79d-48ee-bcdf-b4bd69a1b272)

Reply:
Shouldn't we fix that instead?

Reply:
I'm not technically able to do this, but if we're going to fix, it would also be great to have a way to bring the user timezone value to the Twig template
This way I can update the tip to something like "Based on UTC-3" being the timezone a variable, and say to users to change in their profile 

Could you fix it?

Reply:
Looking at the code the chart widgets are getting the user timezone. Perhaps it's just the table widget?

Reply:
Hm, I tested with other table widgets, all of them use the system timezone
We can start looking at those:
- Recent activity
- Upcoming emails

It might be useful to check the "Best email read hours" widget too _(chart, but just to be sure)_

Reply:
In the class that is generating DB queries for the charts there are some timezone manipulations like https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/Helper/Chart/ChartQuery.php#L585-L586.

Reply:
If I’m not mistaken, table widgets are using a Twig variable related to an extension, probably a way too different solution

Should I wait for you to bring a solution and set this PR as draft?

Reply:
This is far on my todo list. I just don't want to add this information if doesn't apply to all widgets as that would be even more confusing.

Reply:
Ok, I'll keep it closed for now